yarn.lock

```diff
@@ -3268,11 +3268,6 @@
     "@types/undertaker-registry" "*"
     async-done "~1.3.2"
 
-"@types/uuid@^8.3.0":
```

Comment:
Shouldn't `uuid` also be removed here? Was it never in the lock?

Reply:
`uuid` is also used by other packages, like Jest and dtslint (both transitively).